addons/mail/static/src/scss/discuss.scss

```diff
@@ -189,3 +189,19 @@
         }
     }
 }
+
+// ------------------------------------------------------------------
+// Discuss Notification
+// ------------------------------------------------------------------
+
+.o_Notification_Discuss {
+
+    .toast-header {
+        padding: 8px 10px;
+        color: #212529;
+
+        .o_notification_icon {
+            display: none;
+        }
+    }
+}
```

Comment:
This should not be discuss specific, this should be a basic Notification layout/options (to remove icon) if that is want we want in Odoo.

Reply:
I just want to confirm this point as we want to remove icon from notification for generic or for discuss only? As the name of this task is discuss notification design.

Reply:
If we don't want to remove the icon for other notifications (please confirm with NGR), we should at least update the basic notification to allow _an option_ to disable the icon (and therefore not have it in the DOM if possible, using `display: none` is just a hack). It is just not clean to do an override for this with CSS specificity rules, even it it works.
Same reasoning for color, if we should be able to change it, it should be an option.

Also need to double check but I think the padding change was wanted for every notification too?